frontend/lib/src/components/input_field.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
```

Comment:
no need to put this in a folder under components?

`frontend/lib/src/components/input_field/input_field.dart`

Reply:
i asked sir dem and he said its fine for this one

Reply:
@karlolee50